azuredevops/templates/test-run-step-template.yml

```diff
@@ -40,21 +40,19 @@ parameters:
   default: interpret_conda_env
 
 steps:
-- template: conda-path-step-template.yml
-
-- bash: brew install libomp
-  displayName: Install OpenMP with Homebrew
-  condition:  eq(variables['Agent.OS'], 'Darwin')
-
-- bash: |
-      python tools/generate_conda_files.py --name ${{parameters.condaEnv}} --python-version ${{parameters.pythonVersion}}
-      conda env create -n ${{parameters.condaEnv}} --file=${{parameters.condaEnv}}.yaml
-  displayName: 'Creating Conda Environment with dependencies'
-
+- template: conda-setup-step-template.yml
+  parameters:
+    pythonVersion: ${{parameters.pythonVersion}}
+    pinRequirements: ${{parameters.pinRequirements}}
+    envInfoArtifact: ${{parameters.envInfoArtifact}}
+    envInfoFileBase: ${{parameters.envInfoFileBase}}
+    condaEnv: ${{parameters.condaEnv}}
+  
 - bash: |
       source activate ${{parameters.condaEnv}}
-      python --version
-  displayName: 'Verify Python version'
+      python -m pip install flake8
+      flake8 --ignore=E501,E266,W503,F403,F405,E731,F821,E203 ./python/
```

Comment:
why are these ignored here?
--ignore=E501,E266,W503,F403,F405,E731,F821,E203
were they ignored before too?

Reply:
I guess they are based on below.  Were they ignored in the repo before the pipelines changes?

Reply:
I did not change anything about this.